pages/_app.tsx

```diff
@@ -13,6 +15,31 @@ import { ReactComponent as ConceptsIcon } from "../assets/images/component-icons
 import { ReactComponent as SdksToolsIcon } from "../assets/images/component-icons/side-nav/sdks-tools-nav-icon.svg";
 import "react-tippy/dist/tippy.css";
 
+const GoogleTagsAndAnalytics = ({ router }: any) => {
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      TagManager.initialize({ gtmId: "GTM-WRT5LD6" });
+    }
+    ReactGA.initialize("UA-30404064-1");
```

Comment:
Is Google Analytics not set up through GTM?

Reply:
I was thinking maybe yes, but ended up both to the ticket. My fault. Agreed though, Let's see if it can all be managed through GTM!

Reply:
@Finholt , I'm working on adding google analytics so it's managed through GTM now.